packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -77,6 +77,7 @@ export class FetchHttpHandler implements HttpHandler<FetchHttpHandlerConfig> {
     }
     const requestTimeoutInMs = this.config!.requestTimeout;
     const keepAlive = this.config!.keepAlive === true;
+    const credentials = this.config!.credentials as RequestInit["credentials"];
```

Comment:
we're already using `RequestInit`, so `RequestCredentials` is _probably_ globally available too, but I'm being conservative here.